.github/workflows/stale.yaml

```diff
@@ -0,0 +1,24 @@
+name: Close stale PRs
+
+on:
+  schedule:
+  - cron: "00 0 * * *" # runs at 00:00 daily
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v5.1.1
+      name: Clean up stale PRs
+      with:
+        repo-token: ${{ secrets.stale_bot }}
```

Comment:
Are secret names case-sensitive? Looking at the repo settings it seems to be `STALE_BOT` so this might need to match that case?

Reply:
great question, I can't find anything in the docs regarding case sensitivity for secrets for Actions specifically (just for Codespaces, which is [case insensitive](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-github-codespaces#naming-secrets)), so I'll change it to be safe!